app/documents/order_matrix.rb

```diff
@@ -13,22 +13,26 @@ def title
   end
 
   def body
-    order_articles = @order.order_articles.ordered
+    order_articles = @order.order_articles.sort_by{|a| a.article.name.downcase}
```

Comment:
can we do this sorting via ActiveRecord instead?

Reply:
I did the sorting still with sort_by, because I was afraid, that I make unwanted changes in the database with ActiveRecord sorting. Feel free to change this if this is a problem.

Reply:
Sorting belongs in the database, this is a memory hog and doesn't scale for large orders. Please reconsider the `ordered` scope if you think a different sort order is useful everywhere; if it only makes sense here, don't use the `.ordered` scope but use active-record sorting. If you'd like case-insensitive sorting, indexes may also be needed.

Sorry for the condensed answer, if you need more explanation, please don't hesitate to ask. We'll do our best to help!